db/migrations/0002_movie.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.2 on 2024-09-11 09:10
```

Comment:
It is a good practice to include one migration per app per PR in Django projects since it helps to avoid migrations folder overload in the future. Represent all changes done to models in one migration file per app per PR, please